### PR TITLE
aws-cloudwatch: add Korean translation

### PR DESCRIPTION
**Pull request by korECM**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ko/common/aws-cloudwatch.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`aws cloudwatch put-metric-alarm --alarm-name {{알람_이름}} --evaluation-periods {{평가_주기}} --comparison-operator {{비교_연산자}}`
+
+- 지정한 알람들 삭제:
+
+`aws cloudwatch delete-alarms --alarm_names {{alarm_names}}`
+
+- 지정한 대시보드들 삭제:
+
+`aws cloudwatch delete-dashboards --dashboard-names {{dashboard_names}}`
```
Comment (kbdharun) [inline pages.ko/common/aws-cloudwatch.md:32]:
```suggestion
`aws cloudwatch delete-dashboards --dashboard-names {{대시보드_이름}}`
```

File: pages.ko/common/aws-cloudwatch.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`aws cloudwatch describe-alarms`
+
+- 해당 매트릭과 연결된 알람 생성(또는 업데이트):
+
+`aws cloudwatch put-metric-alarm --alarm-name {{알람_이름}} --evaluation-periods {{평가_주기}} --comparison-operator {{비교_연산자}}`
+
+- 지정한 알람들 삭제:
+
+`aws cloudwatch delete-alarms --alarm_names {{alarm_names}}`
```
Comment (kbdharun) [inline pages.ko/common/aws-cloudwatch.md:28]:
```suggestion
`aws cloudwatch delete-alarms --alarm_names {{알람_이름}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page. (Btw, I referred to your other translated placeholders)

**Comment by korECM** [comment]
@kbdharun 
I have accepted all of the suggestions you provided! Thank you as always!

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**